homeassistant/components/alexa/smart_home.py

```diff
@@ -66,6 +75,10 @@
 ENTITY_ADAPTERS = Registry()
 EVENT_ALEXA_SMART_HOME = 'alexa_smart_home'
 
+ALEXA_URI = "https://api.amazonalexa.com/v3/events"
```

Comment:
This depends on the region: https://developer.amazon.com/de/docs/smarthome/send-events-to-the-alexa-event-gateway.html#endpoints

We should get both the endpoint URI and auth from the alexa config object that we pass around.